server/src/main/java/org/opensearch/index/mapper/DocumentParser.java

```diff
@@ -695,9 +698,10 @@ private static void parseNullValue(ParseContext context, ObjectMapper parentMapp
         if (mapper != null) {
             // TODO: passing null to an object seems bogus?
             parseObjectOrField(context, mapper);
-        } else if (parentMapper.dynamic() == ObjectMapper.Dynamic.STRICT) {
-            throw new StrictDynamicMappingException(parentMapper.fullPath(), lastFieldName);
-        }
+        } else if (parentMapper.dynamic() == ObjectMapper.Dynamic.STRICT
```

Comment:
Super nit, it would be good to extract `parentMapper.dynamic()` to local variable, we use it like 3 times there